src/radical/entk/appman/config.json

```diff
@@ -11,6 +9,7 @@
                          "db_cleanup"      : false },
     "pending_qs"      : 1,
     "completed_qs"    : 1,
-    "rmq_cleanup"     : true
+    "rmq_cleanup"     : true,
+    "rmq_url"         : "amqp://guest:guest@localhost:5672/%2F"
```

Comment:
Is the `%2F` intentional?

Reply:
Yes, it's a URL-encoded "/", which is the default virtual host. I don't understand exactly why it needs to be URL-encoded, but that's what the Pika documentation said to do.